app/Http/Controllers/PermissionGroup/PermissionGroupController.php

```diff
@@ -3,19 +3,24 @@
 namespace App\Http\Controllers\PermissionGroup;
 
 use App\Http\Controllers\Controller;
-use Illuminate\Http\Request;
-use 
+use App\Http\Requests\PermissionGroup\PermissionGroupRequest;
+use App\Repositories\Admin\PermissionGroup\PermissionGroupRepository;
 
 class PermissionGroupController extends Controller
 {
-    /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
+    private $permissionGroupRepository;
+
+    public function __construct(PermissionGroupRepository $permissionGroupRepository)
+    {
+        $this->permissionGroupRepository = $permissionGroupRepository;
+    }
+
     public function index()
     {
-        //
+        $permissionGroup = $this->permissionGroupRepository->paginate();
+        return view('admin.permission-group.index', [
+            'permissionGroup' => $permissionGroup,
```

Comment:
'permissionGroup' => $permissionGroup,
=>
 'permissionGroups' => $this->permissionGroupRepository->paginate(),